[2025-09-12T07:15:51.152+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-12T07:15:51.228+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_hourly_predictions.check_system manual__2025-09-12T06:51:45.475730+00:00 [queued]>
[2025-09-12T07:15:51.242+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_hourly_predictions.check_system manual__2025-09-12T06:51:45.475730+00:00 [queued]>
[2025-09-12T07:15:51.245+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-12T07:15:51.277+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_system> on 2025-09-12 06:51:45.475730+00:00
[2025-09-12T07:15:51.307+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_hourly_predictions', 'check_system', 'manual__2025-09-12T06:51:45.475730+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/tmp/tmpbrhja1ep']
[2025-09-12T07:15:51.317+0000] {standard_task_runner.py:91} INFO - Job 54: Subtask check_system
[2025-09-12T07:15:51.319+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=199) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-12T07:15:51.329+0000] {standard_task_runner.py:64} INFO - Started process 232 to run task
[2025-09-12T07:15:51.544+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_hourly_predictions.check_system manual__2025-09-12T06:51:45.475730+00:00 [running]> on host 2058836b81fe
[2025-09-12T07:15:53.254+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ops-team' AIRFLOW_CTX_DAG_ID='simple_hourly_predictions' AIRFLOW_CTX_TASK_ID='check_system' AIRFLOW_CTX_EXECUTION_DATE='2025-09-12T06:51:45.475730+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-12T06:51:45.475730+00:00'
[2025-09-12T07:15:53.264+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-12T07:15:53.458+0000] {logging_mixin.py:188} INFO - üîç Quick system check...
[2025-09-12T07:15:53.460+0000] {logging_mixin.py:188} INFO - ‚úÖ Disk space OK: 924.6GB
[2025-09-12T07:15:53.496+0000] {logging_mixin.py:188} INFO - ‚ö†Ô∏è API not reachable
[2025-09-12T07:15:53.497+0000] {python.py:237} INFO - Done. Returned value was: system_checked
[2025-09-12T07:15:53.498+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-12T07:15:53.791+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_hourly_predictions, task_id=check_system, run_id=manual__2025-09-12T06:51:45.475730+00:00, execution_date=20250912T065145, start_date=20250912T071551, end_date=20250912T071553
[2025-09-12T07:15:54.125+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-12T07:15:54.314+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-12T07:15:54.522+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
